src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -124,8 +108,8 @@ tests:
 
       # verification: ""
     - label:
-          "Step 6.2: TH2 generates the NOC, the Root CA Certificate and ICAC
-          using the CSR elements from Step 5 and selects an IPK, all for use by
+          "Step 4.1: TH2 generates the NOC, the Root CA Certificate and ICAC
```

Comment:
```suggestion
          "Step 4: TH2 generates the NOC, ICAC, and IPK for its fabric
```

Reply:
Fixed: I updated the step from 4.1 to 5.1, as Step 4 was updated to Step 5 and the sequence was modified.